setup.py

```diff
@@ -45,7 +45,8 @@ def get_version():
             "django_add_default_value>=0.4.0",
             'mock>=3.0.5;python_version<"3.3"',
             'backports.tempfile>=1.0;python_version<="2.7"',
-        ]
+        ],
+        "toml": ["toml==0.10.2"],
```

Comment:
Please avoid using strict version definitions, we should prefer using lower bounds for dependencies
```suggestion
        "toml": ["toml>=0.10.2"],
```